src/web/components/elements/InstagramBlockComponent.tsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
 
-import { unescapeData } from '@root/src/lib/escapeData';
 import { css } from 'emotion';
 
-const widthOverride = css`
-	iframe {
-		/* The instagram embed js hijacks the iframe and calculated an incorrect width, which pushed the body out */
-		min-width: 300px !important;
-	}
+const fullWidthStyles = css`
+	width: 100%;
```

Comment:
What was the use case for having this before. Will this still solve that?

Reply:
I think previously there was an issue where the instagram js was creating an iframe that was too big for the article.

In this new approach I think the instagram code will be 'trapped' inside the our iframe which this sets to 100% width.

I guess in the event instagram creates something that is too big it would get clipped, which isnt ideal but wouldnt break anything outside the component.

TBH Ive never been able to reproduce anything like that. 

Also a bunch of Instagram embeds get rendered like this all the time with no complaints, ie the ones in 'embed' type elements.

Reply:
I think you're right, now that we create our own iframe we should maintain control. In fact, I wonder if we need any styling here?

Reply:
Without this stying the iframe defaults to ~300px wide so I think we do need this.